### ISSUE TITLE
helm.v3.Release throws error Chart.yaml file is missing for Dapr chart

### ISSUE THREAD

**Issue by jancespivo** [pulumi/pulumi-kubernetes#1719]
Steps to reproduce:

```python
kubernetes.helm.v3.Release(
    "dapr",
    kubernetes.helm.v3.ReleaseArgs(
        chart="dapr",
        version="1.4.1",
        repository_opts=kubernetes.helm.v3.RepositoryOptsArgs(
            repo="https://dapr.github.io/helm-charts/",
        ),
        values={},
    )
)
```

throws error:
```
kubernetes:helm.sh/v3:Release (dapr):
    error: Chart.yaml file is missing
```

`helm upgrade --install -n dapr-system dapr dapr/dapr --version=1.4.0` works properly (if crds are already there)

**Comment by jancespivo** [comment]
I tried to change the chart from `dapr` to `dapr/dapr` and it started to work. But it seems `dapr/dapr` works well only if repo is installed locally, otherwise it fails with:
`error: chart "dapr/dapr" version "1.4.1" not found in https://dapr.github.io/helm-charts/ repository`
So the origin issue is still there.

**Comment by viveklak** [comment]
```
helm repo add dapr https://dapr.github.io/helm-charts/
helm repo update
helm search repo dapr
NAME            CHART VERSION   APP VERSION     DESCRIPTION                        
dapr/dapr       1.4.0           1.4.0           A Helm chart for Dapr on Kubernetes
```

```
const prometheusRelease = new k8s.helm.v3.Release("dapr", {
    name: "dapr",
    chart: "dapr",
    version: "1.4.0",
    repositoryOpts: {
        repo: "https://dapr.github.io/helm-charts/",
    },
    values: {},
});
```
Works fine for me. Note that a release is imminent with fixes in #1718 which will resolve issues for charts with CRDs.

**Comment by jancespivo** [comment]
@viveklak Thx! I have a `dapr` folder inside my pulumi project and Helm looks into that folder instead of using a repo!
